docs/com/smartcar/sdk/AuthClient.html

```diff
@@ -336,8 +336,8 @@ <h3>Method Summary</h3>
 </tr>
 <tr id="i15" class="rowColor">
 <td class="colFirst"><code>boolean</code></td>
-<td class="colLast"><code><span class="memberNameLink"><a href="../../../com/smartcar/sdk/AuthClient.html#isCompatible-java.lang.String-">isCompatible</a></span>(java.lang.String&nbsp;vin)</code>
-<div class="block">Determine if a vehicle is compatible with Smartcar.</div>
+<td class="colLast"><code><span class="memberNameLink"><a href="../../../com/smartcar/sdk/AuthClient.html#isCompatible-java.lang.String-">isCompatible</a></span>&#8203;(java.lang.String&nbsp;vin&nbsp;String[]&nbsp;scope)</code>
```

Comment:
can you match these from https://github.com/smartcar/api-reference/pull/93, please